symplyphysics/core/symbols/symbols.py

```diff
@@ -37,7 +37,7 @@ def display_latex(self) -> str:
         return self._display_latex
 
     def _sympystr(self, p: Printer) -> str:
-        return str(p.doprint(self.display_name))
+        return p.doprint(self.display_name)  # type: ignore[no-any-return]
```

Comment:
It seems Printer does not have type hints, that's why it assumes Any here. So here you can forcefully convert to str()